ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/CommandDeduplicationIT.scala

```diff
@@ -468,6 +458,93 @@ final class CommandDeduplicationIT(
     } yield {}
   })
 
+  testGivenAllParticipants(
+    shortIdentifier = s"DeduplicateUsingDurations",
+    description = "Deduplicate commands within the deduplication period defined by a duration",
+    participants = allocate(SingleParty),
+    timeoutScale = 2, // has to wait for the deduplication period to expire
```

Comment:
Is this required? Isn't the dedup period < than 30 seconds?

Reply:
At some point, one of the tests was timing out. I couldn't reproduce the timeout, so I deleted it.